tools/cldr-apps/src/main/webapp/js/esm/cldrSurvey.js

```diff
@@ -725,65 +733,6 @@ function getTagChildren(tr) {
   return rowChildren;
 }
 
-/**
- * Wire up the button to perform a submit
- *
- * @param button
```

Comment:
The rest of this PR is just moving some functions from cldrSurvey.js to the new cldrVote.js and fixing the exports and namespaces accordingly